examples/frigate/frigate.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+def frigate_events(topic, data, srv=None):
+    if srv is not None:
+        srv.logging.debug('events ******************************************')
```

Comment:
You can probably remove these debug messages.

Reply:
Addressed with c2623646b, thanks.